examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
Also, changing it in this way contradicts the `README`: https://github.com/bokeh/bokeh/blob/master/examples/models/README.md

Currently the only expect place to find "apps" is in `examples/apps` and I'd very much like to keep that. So my preference is that this be reverted, or moved to `examples/apps`